src/controllers/users.controllers.js

```diff
@@ -88,6 +88,23 @@ export default class Users {
     });
   }
 
+  static async setRole(req, res) {
+    try {
+      const { role } = req.body;
+      const { email } = req.params;
+      await User.setRole(role, email);
```

Comment:
Good job here! But it seems that you have missed this part of the story:

Story: Admin should be able to assign various users to various roles. There should also be a role for the admin account itself. **This will allow only accounts with particular roles permission to perform particular actions.**

- [x] An admin can assign roles
- [ ] How will an admin permit this role to perform particular actions?